File: 기초 사고력문제/백준_1924_2007년/2007년.java
import java.util.*;

public class Main{

    public static void main(String[] args){
        int[] month = {31,28,31,30,31,30,31,31,30,31,30,31};
        String[] week = {"MON", "TUE", "WED", "THU", "FRI", "SAT", "SUN"};

        Scanner s = new Scanner(System.in);
        int m = s.nextInt();
        int d = s.nextInt();

        int sum = 0;
        for(int i = 0; i< m-1; i++) sum += month[i];
        sum += (d-1);
        System.out.println(week[sum % 7]);
    }
}

File: 기초 사고력문제/백준_1924_2007년/2007년.py
m,d = map(int, input().split())
month = [31,28,31,30,31,30,31,31,30,31,30,31]
week = ['MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT', 'SUN']

sum = 0

for i in range(m-1):
    sum += month[i]
sum += (d-1)
print(week[sum % 7])